express-server/constants/HTTPStatus.js

```diff
@@ -1,4 +1,8 @@
-export const HTTPStatuses = {
+const HTTPStatuses = {
```

Comment:
why did u change all the exports?

Reply:
because when i set the type of my folder to be "module" i could use es6 import/export, but it messed up many things, i.e. i couldn't use __dirname and other nodejs stuff, so i just erased the "module" type from package json and all went back to normal

Reply:
latest node version supports import / export. plz fix.

Reply:
i'll install the latest nodejs version